nemo/collections/common/prompts/canary.py

```diff
@@ -16,9 +16,9 @@ class CanaryPromptFormatter(PromptFormatter):
             "template": f"{CANARY_BOS}|source_lang||task||target_lang||pnc|",
             "slots": {
                 "source_lang": Modality.Text,
-                "task": Modality.Text,
+                "task": Modality.TextLiteral("asr", "asr", "s2t_translation", "<|transcribe|>", "<|translate|>"),
```

Comment:
typo, two times "asr" instead of "asr", "ast"

Reply:
Very good catch